packages/shared/src/components/notifications/NotificationItem.tsx

```diff
@@ -24,6 +24,17 @@ function NotificationItem({
   attachments,
 }: NotificationItemProps): ReactElement {
   const purify = useDomPurify();
+  const { title: memoizedTitle, description: memoizedDescription } =
+    useMemo(() => {
+      if (!purify?.sanitize) {
+        return { title: '', description: '' };
+      }
+
+      return {
+        title: purify.sanitize(title),
+        description: purify.sanitize(description),
+      };
+    }, [purify]);
```

Comment:
```suggestion
    }, [purify, title, description]);
```

Reply:
Oh, just saw this now. The most recent push should have this covered. Thank you!